src/main/java/com/diffplug/gradle/pde/PdeAntBuildTask.java

```diff
@@ -43,6 +45,17 @@
  * ```
  */
 public class PdeAntBuildTask extends DefaultTask {
+	WorkspaceRegistry registry;
+
+	public PdeAntBuildTask() {
+		try {
+			this.registry = WorkspaceRegistry.instance();
+		} catch (IOException e) {
+			getLogger().warn("WorkspaceRegistry could not be instantiated! Using default workspace.", e);
+			this.registry = null;
+		}
+	}
+
 	private Object antFile;
```

Comment:
I try to avoid state and config whenever possible. I would remove all this config, and just assume that WorkspaceRegistry.instance() will always succeed.